addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6436,6 +6499,14 @@ registry.ImageTools = ImageHandlerOption.extend({
                 const img = this._getImg();
                 return (img.dataset.shapeColors && img.dataset.shapeColors.split(';')[parseInt(params.colorId)]) || '';
             }
+            case 'setImgShapeFlipX': {
+                const img = this._getImg();
+                return (img.dataset.shapeFlip && img.dataset.shapeFlip.includes("x")) || "";
```

Comment:
Note that this would be a good case to use the new syntax:
```suggestion
                return img.dataset.shapeFlip?.includes("x") || "";
```